tf_trees/README.md

```diff
@@ -1,18 +1,47 @@
 # The Tree Ensemble Layer
 
-This repository contains a new *tree ensemble layer* which can be used anywhere in a neural network. We provide a low-level Tensorflow implementation along with a high-level Keras API.
+This repository contains a new *tree ensemble layer* (TEL) for neural networks. The layer is differentiable so SGD can be used to train the neural network (including TEL). The layer supports conditional computation for both training and inference, i.e., when updating/evaluating a certain node in the tree, only the samples that reach that node are used in computations (this is to be contrasted with the dense computations in neural networks). We provide a low-level TensorFlow implementation along with a high-level Keras API.
 
-The layer is differentiable so SGD can be used to train the neural network (including the tree layer). The layer supports conditional computation for training and inference: when updating/evaluating a certain node in the tree, only the samples that reach that node are used in computations (this is to be contrasted with the dense computations in neural networks).
+More details to be added soon.
+
+## Installation
+The installation instructions below assume that you have Python and TensorFlow already installed. Use Method 1 if you have installed TensorFlow from source. Otherwise, use Method 2.
+
+### Method 1: Compile using Bazel
+First, copy the file "BUILD" (available in the tf_trees directory) to the directory "tensorflow/core/user_ops".
+Then, run the following command:
+```bash
+bazel build --config opt //tensorflow/core/user_ops:neural_trees_ops.so
```

Comment:
@tanzhenyu

Reply:
Let's say "If you have TensorFlow sources installed, you can make use of TensorFlow's build system to compile your op."